Documentation/MobileCore/api-reference.md

```diff
@@ -130,8 +130,101 @@ final Application app = MobileCore.getApplication();
 val app = MobileCore.getApplication()
 ```
 
+#### Initializing the SDK by automatically registering all extensions and enabling Lifecycle data collection
 
-#### Registering extensions and starting the SDK
+> [!NOTE]
+> API `initialize(String appId)` added since Core 3.3.0
+
+> [!IMPORTANT]
+> All Adobe Mobile SDK extensions listed as a dependency in your application are automatically registered when calling `initialize(String appId)`.
+>
+> Automatic Lifecycle data collection requires **Lifecycle** extension included as an app dependency.
+
+##### Java
+
+```java
+public class YourApp extends Application {
+
+   @Override
+   public void onCreate() {
+      super.onCreate();
+
+      MobileCore.initialize(this, YOUR_APP_ID);
+      
+      // Optionally, if you need a callback:
+      // MobileCore.initialize(this, YOUR_APP_ID, new AdobeCallback<Object>() {
+      //     // SDK initialized.
```

Comment:
Do you need to include for a more complete example?
```        
@Override
public void call(Void result) {
   // SDK initialized.
}
```

Reply:
No, we have a similar snippet for all the other callbacks. Do you have anything in mind?